client/lib/feed-stream-store/actions.js

```diff
@@ -117,20 +118,20 @@ export function receivePage( id, error, data ) {
 }
 
 export function receiveUpdates( id, error, data ) {
-	if ( ! error && data && data.posts ) {
+	// if conversations, then update the comments on the post
+	if ( data.posts && data.posts[ 0 ] && data.posts[ 0 ].comments ) {
 		forEach( data.posts, post => {
-			if ( post.comments ) {
-				// conversations!
-				reduxDispatch( {
-					type: COMMENTS_RECEIVE,
-					siteId: post.site_ID,
-					postId: post.ID,
-					comments: post.comments,
-				} );
-			}
+			const postKey = { blogId: post.site_ID, postId: post.ID };
```

Comment:
this may have unintended consequences... why was it added?

Reply:
before the receiveUpdates the post has old comments (i think). with this I update it to have the new comments so that it can be received on acceptance

Reply:
afaik the comments should only live on the postKey in the stream, not on the actual post object in the post store...

Reply:
discussed in slack and agreed on a different strategy where we create `pendingComments`